src/sidebar/services/persisted-defaults.js

```diff
@@ -46,7 +51,7 @@ export class PersistedDefaultsService {
     Object.keys(DEFAULT_KEYS).forEach(defaultKey => {
       // `localStorage.getItem` will return `null` for a non-existent key
       const defaultValue = this._storage.getItem(DEFAULT_KEYS[defaultKey]);
-      this._store.setDefault(defaultKey, defaultValue);
+      this._store.setDefault(/** @type {Key} */ (defaultKey), defaultValue);
```

Comment:
`Object.keys(T)` returns `string[]` rather than `keyof T[]`. I think this is for historical reasons.